.github/workflows/test.yaml

```
@@ -23,7 +23,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        goversion: ['1.20', '1.21']
+        goversion: ['1.22', '1.21']
```

Nit: ascending order perhaps? Or is that the appropriate convention?

```suggestion
        goversion: ['1.21', '1.22']
```